### ISSUE TITLE
Reduced circulating interferon gamma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6462]
**Preferred term label:**
Reduced circulating interferon gamma

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4836962/#SD2

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:301220 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
